pom.xml

```diff
@@ -428,6 +428,20 @@
                 <version>${version.org.wildfly.core}</version>
                 <type>pom</type>
                 <scope>import</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.bouncycastle</groupId>
+                        <artifactId>bcpg-jdk15on</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.bouncycastle</groupId>
+                        <artifactId>bcpkix-jdk15on</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.bouncycastle</groupId>
+                        <artifactId>bcprov-jdk15on</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
What's needed is not the exclusion of these dependencies from core, but rather removal of these dependencies from full. Core will provide these libraries so full should remove everything related to providing them. 

1) They should be removed from this file's dependencyManagement as https://github.com/wildfly/wildfly/pull/12791/files#diff-600376dffeb79835ede4a0b285078036R430 brings them in via import of core's pom.
2) They should be removed from https://github.com/wildfly/wildfly/blob/master/feature-pack/pom.xml as that legacy feature pack no longer is their source.
3) They should be removed from https://github.com/wildfly/wildfly/blob/master/galleon-pack/pom.xml as that galleon feature pack is no longer their source.
4) They should be removed from https://github.com/wildfly/wildfly/blob/master/feature-pack/src/license/full-feature-pack-licenses.xml as the full f-ps are no longer providing their licenses.
5) The module dir at https://github.com/wildfly/wildfly/tree/master/feature-pack/src/main/resources/modules/system/layers/base/org/bouncycastle/main should be removed as the core feature pack provides the module.

Reply:
Thank you for the feedback! I've updated the PR to remove `org.bouncycastle:bcprov-jdk15on` and `org.bouncycastle:bcpkix-jdk15on` from WildFly as these would be imported from WildFly Core. `org.bouncycastle:bcmail-jdk15on` is not used in Core so I've kept it in WildFly for now. 

When adding these changes, I got an error that `bcprov-jdk15on` was being brought in as a transitive dependency by `org.jboss.resteasy:jose-jwt`, so I've added it as an exclusion here: https://github.com/wildfly/wildfly/blob/b3d87cfc7bb484961ba7e92dbde7d24b708ab7e4/pom.xml#L6624 
Running the full WildFly testsuite with these changes and my WildFly core changes (wildfly/wildfly-core#249) was successful. cc: @asoldano